fiftyone/utils/patches.py

```diff
@@ -45,12 +46,21 @@ class ImagePatchesExtractor(object):
     def __init__(
         self,
         samples,
-        patches_field,
+        patches_field=None,
         include_labels=False,
         force_rgb=False,
         force_square=False,
         alpha=None,
     ):
+        if patches_field is None:
+            if samples._is_patches:
+                patches_field = samples._label_fields[0]
```

Comment:
In the situation where other label fields are copied into the patches view, will this always assign the correct patches field?

Reply:
yep `_label_fields` is a property specific to patches views that just contains the field(s) that defined the view.